test/common/performance/receiver/id_extractor.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package receiver
+
+import cloudevents "github.com/cloudevents/sdk-go"
+
+type IdExtractor func(event cloudevents.Event) string
+
+func EventIdExtractor(event cloudevents.Event) string {
+	return event.ID()
```

Comment:
Do we anticipate other ways to extract an ID?

Reply:
In source tests, the id and type are sent inside the payload, so a different logic to extract it is required (eg in kafka source event id and event type are configured by the kafka source itself and doesn't depend on the message)

Reply:
Ok thanks for the explanation.
Could we maybe have both extractors in a single file with a small godoc which explains that to future contributors?